CHANGELOG.md

```diff
@@ -2,12 +2,18 @@
 
 ## Next version
 
-- Put your changes here...
+- Fixed a bug where views bundler feature could fail to load in some rare instances.
+- Updated various dependencies.
 
 ## 0.24.0
 
+- Breaking: `startServer` and `initServer` methods are now async and need to be awaited to avoid possible race conditions.
```

Comment:
```suggestion
- Breaking: Altered `startServer` and `initServer` methods to now be async. They now need to be awaited to avoid possible race conditions.
```